tools/scripts/lint_sh.sh

```diff
@@ -8,8 +8,24 @@ list_sh_src() {
     git ls-files -z -c -m -o --exclude-standard -- '*.sh' '*.bash'
 }
 
+
+unameOut="$(uname -s)"
+case "${unameOut}" in
+    Linux*)     platform="linux";;
+    Darwin*)    platform="mac";;
+    *)          exit 1
```

Comment:
This exits the whole script, right? An error message might be nice, I s'pose, or we could try the system shellcheck in that case.

Reply:
I just copied the boilerplate we have in all our other scripts but i can change all of these in a follow up if you'd like